src/cluster.py

```diff
@@ -256,6 +253,8 @@ def cluster_status(
     ) -> Optional[list[ClusterMember]]:
         """Query the cluster status."""
         # Request info from cluster endpoint (which returns all members of the cluster).
+        # TODO we don't know the other cluster's ca
+        verify = self.verify if not alternative_endpoints else False
```

Comment:
With async replication (when we add the alternative endpoints of the other cluster), we don't know the other side's CA, so can't really verify the cert at the moment. We should add the CAs to the async relations so that we can verify the certs as needed.